kuwala/pipelines/google-poi/resources/categories.json

```diff
@@ -0,0 +1,205 @@
+{
+    "administration": {
+        "category": "administration",
+        "tags": [
+            "association_or_organization",
+            "city_hall"
+        ]
+    },
+    "airport": {
+        "category": "airport",
+        "tags": [
+            "international_airport",
+            "airport"
+        ]
+    },
+    "apartment": {
+        "category": "apartment",
+        "tags": [
+            "furnished_apartment_building"
+        ]
+    },
+    "art_culture": {
```

Comment:
JSON key should still be camel case due to JSON naming convention. (See previous conversation)

Reply:
ok. :) I will check the conversation and work on this later.